Makefile

```diff
@@ -250,6 +250,14 @@ cross:
 $(notdir $(abspath $(wildcard cmd/*/))): generated_files
 	hack/make-rules/build.sh cmd/$@
 
+# Add rules for kube-scheduler
+#
+# Example:
+#   make kube-scheduler
+.PHONY: kube-scheduler
+kube-scheduler: generated_files
+	hack/make-rules/build.sh plugin/cmd/kube-scheduler
```

Comment:
That's fine that there is only one dir there, for now.  Those things tend
to change :)

On Thu, Jul 28, 2016 at 11:10 PM, Peter Zhao <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/kubernetes/kubernetes/pull/29733#discussion_r72745516>
> :
>
> > @@ -250,6 +250,14 @@ cross:
> >  $(notdir $(abspath $(wildcard cmd/*/))): generated_files
> >  	hack/make-rules/build.sh cmd/$@
> >
> > +# Add rules for kube-scheduler
> > +#
> > +# Example:
> > +#   make kube-scheduler
> > +.PHONY: kube-scheduler
> > +kube-scheduler: generated_files
> > +	hack/make-rules/build.sh plugin/cmd/kube-scheduler
>
> I saw that there is only kube-scheduler in plugin/cmd/, so I just put it
> directly here instead of the way you mentioned. Surely your way is
> compatible.
>
> Is it OK as below?
>
> .PHONY: $(notdir $(abspath $(wildcard plugin/cmd/*/)))
> $(notdir $(abspath $(wildcard plugin/cmd/*/))): generated_files
>     hack/make-rules/build.sh plugin/cmd/$@
>
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29733/files/6009073d3042f9070e350cedac848c92ca9b85f9#r72745516>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVHHgm9_WaXm1mcnju3Agck5aajTPks5qaZlegaJpZM4JXJO9>
> .
>